RELEASE_NOTES.md

```diff
@@ -8,6 +8,8 @@ Release notes template:
 
 ## Fixed
 
+* Email addresses for external users no longer have "@princeton.edu" added to the end of valid email addresses.
```

Comment:
I think you added this to the documentation template instead of as a new entry.

Reply:
😞 yes, that's exactly what I did.